File: aat-android/src/main/java/ch/bailu/aat/description/AveragePaceDescription.java
package ch.bailu.aat.description;

import ch.bailu.aat_lib.gpx.GpxInformation;
import ch.bailu.aat_lib.preferences.StorageInterface;
import ch.bailu.aat_lib.resources.Res;

public class AveragePaceDescription extends PaceDescription {
    public AveragePaceDescription(StorageInterface s) {
        super(s);
    }



    @Override
    public String getLabel() {
        return Res.str().pace();
    }

    @Override
    public void onContentUpdated(int iid, GpxInformation info) {
        setCache(speedToPace(info.getSpeed()));
    }

}

File: aat-lib/src/main/java/ch/bailu/aat_lib/description/AveragePaceDescription.kt
package ch.bailu.aat_lib.description

import ch.bailu.aat_lib.gpx.GpxInformation
import ch.bailu.aat_lib.preferences.StorageInterface
import ch.bailu.aat_lib.resources.Res

open class AveragePaceDescription(s: StorageInterface) : PaceDescription(s) {
    override fun getLabel(): String {
        return Res.str().pace()
    }

    override fun onContentUpdated(iid: Int, info: GpxInformation) {
        setCache(speedToPace(info.getSpeed()))
    }
}
